apps/vr-tests-react-components/src/stories/TimePicker.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { TimePicker } from '@fluentui/react-timepicker-compat-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('TimePicker compat', module)
```

Comment:
curious why do we allow to use long deprecated story format  ?  I see mix/match of this within our VR test stories. 

@ling1726 @TristanWatanabe  

lets rewrite this to CSF format

Reply:
Agreed, new vr tests should be implemented with CSF. You can use the following as an example

* https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/MessageBar/MessageBar.stories.tsx
* https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Toolbar/Toolbar.stories.tsx
* https://github.com/microsoft/fluentui/blob/master/apps/vr-tests-react-components/src/stories/Toast/Toast.stories.tsx

Reply:
Updated :)